packages/app/src/app/overmind/namespaces/dashboard/utils.ts

```diff
@@ -22,7 +22,19 @@ export function sortByNameAscending(
   a: ProjectFragment,
   b: ProjectFragment
 ): number {
-  const repoA = a.repository.owner + '/' + a.repository.name;
-  const repoB = b.repository.owner + '/' + b.repository.name;
-  return repoA < repoB ? -1 : 1;
+  return a.repository.name.toLowerCase() < b.repository.name.toLowerCase()
+    ? -1
+    : 1;
+}
+
+export function getProjectUniqueKey({
```

Comment:
What is the reason we're calling this `project` instead of `repository`, to make a distinction between GitHub and CSB repositories? I think BEOPS does this too, right? Should we define this somewhere e.g. in the glossary?

Reply:
Yes, and to avoid `repository.repository` when accessing the data. I can add this to the glossary